### PR TITLE
WT-6824 Resolve UTF related issues for OS/X.

### PR DESCRIPTION
**Pull request by ddanderson**
For 'wt dump -j' (JSON dumping), never try to print non-ASCII characters.
Relying solely on isprint() to determine printability is a mistake for our
purposes, that function is dependent on local for non-ASCII characters.
In test code, don't use the Python Unicode decoder except with printable chars.

### CODE REVIEW

**Comment by ddanderson** [comment]
@keitharnoldsmith, could you please take a look at this?  If you're busy with the conference, it can be done next week - nobody's really waiting on it.

File: test/suite/test_util01.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-import string, os, sys, random
+import string, os, sys, random, string
```
Comment (keitharnoldsmith) [inline test/suite/test_util01.py:29]: I assume. you don't mean to (or need to) import the string library twice.

**Review by keitharnoldsmith — APPROVED** [review]
One minor comment, but otherwise LGTM

File: test/suite/test_util01.py (hunk 1 above)
Comment (ddanderson) [reply test/suite/test_util01.py]: Fixed.

**Pull request merged**